lib/dialects/abstract/query-interface.js

```diff
@@ -949,11 +949,12 @@ class QueryInterface {
 
   async select(model, tableName, optionsArg) {
     const options = { ...optionsArg, type: QueryTypes.SELECT, model };
+    const query = this.queryGenerator.selectQuery(tableName, options, model);
+    if (options.getRawSql) {
```

Comment:
I'm unsure if `getRawSql` is the best name for this option, I would probably go with something like `queryOnly`, `getQuery`, `query` or something else.